x-pack/plugins/observability/public/pages/overview/components/sections/logs/logs_section.tsx

```diff
@@ -142,7 +142,11 @@ export function LogsSection({ bucketSize }: Props) {
           showLegend
           legendPosition={Position.Right}
           xDomain={{ min, max }}
-          showLegendExtra
+          // Please double check if the data passed to the chart contains all the buckets, even the empty ones.
+          // the showLegendExtra will display the last element of the data array as the default legend value
+          // and if empty buckets are filtered out you can probably see a value that doesn't correspond
+          // to the value in the last time bucket visualized.
+          // showLegendExtra
```

Comment:
if I'm not wrong the query for this chart is built in `buildLogOverviewAggregations` from `x-pack/plugins/infra/public/utils/logs_overview_fetchers.ts` and this doesn't contain the `min_doc_count:0` and the `extended_bounds`. Turning this off for now, but can be turned as soon as the query builder is fixed